index.bs

```diff
@@ -310,7 +310,23 @@ dictionary ClearDataTypeSet {
 <!-- ====================================================================== -->
 
 <!-- ====================================================================== -->
-## Web Request API ## {#api-web-request}
+## Integration with other specifications ## {#api-integration-with-other-specifications}
```

Comment:
I'd intended to upload this with the earlier PR but I hadn't committed my working copy change.